.gitignore

```diff
@@ -7,6 +7,7 @@ config_*.sh
 clouds.yaml
 _clouds_yaml
 metal3-dev/deployment-*.yaml
+custom-images.json
```

Comment:
Seems a bit weird to add this to gitignore when it's not actually part of the script?

Reply:
Removed